docs/conf.py

```diff
@@ -33,6 +33,21 @@
 
 html4_writer = True
 
+# We use these *local* environment variables for private info like free ticket links
+
+cfp_variables = {}
+cfp_variables['upload'] = os.environ.get('WTD_CFP_UPLOAD')
+cfp_variables['ticket'] = os.environ.get('WTD_CFP_SPEAKER_TICKET')
+cfp_variables['calendly'] = os.environ.get('WTD_CFP_CALENDLY')
+cfp_variables['feedback_form'] = os.environ.get('WTD_CFP_FEEDBACK_FORM')
+cfp_variables['speaker_gift'] = os.environ.get('WTD_CFP_SPEAKER_GIFT')
+
+if None not in cfp_variables.values():
+    print('Private CFP environment variables set. ✅')
+    cfp_variables['print_templates'] = True
+else:
+    print('Private CFP environment variables not set. ❌')
```

Comment:
```suggestion
    print('Private CFP environment variables not set, not building CFP email templates.')
```
I think the ❌ emoji might scare people for an otherwise fine and common build scenario.

Reply:
:+1: